tracks/models.py

```diff
@@ -3,58 +3,38 @@
 import uuid
```

Comment:
I removed migrations from git because it got easily out of sync and difficult to debug (different migrations state for local/dev/live database). For now I keep live database migration files in local computer and restart migrations for dev db when models change in review app.

Reply:
I'll add migrations for live db to git.